### DISCUSSION TITLE
Welcome to Tailwind Mobile Discussions!

### DISCUSSION THREAD

**Discussion by nolimits4web** [konstaui/konsta#6] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by VoiceOfSoftware** [comment]
Not sure if this is the right place to ask this question: I'm building a large site with SvelteKit and DaisyUI for desktop browsers. I'd love to add a mobile interface using Konsta to the same codebase. Is it just a matter of installing Konsta into the same project, and using a different route, like /mobile for the Konsta pages? I don't need identical functionality; I just want to know if I can start building a few Konsta pages into my site without breaking the desktop DaisyUI parts.

**Comment by jkhaui** [comment]
> Not sure if this is the right place to ask this question: I'm building a large site with SvelteKit and DaisyUI for desktop browsers. I'd love to add a mobile interface using Konsta to the same codebase. Is it just a matter of installing Konsta into the same project, and using a different route, like /mobile for the Konsta pages? I don't need identical functionality; I just want to know if I can start building a few Konsta pages into my site without breaking the desktop DaisyUI parts.

This sounds fine to me. Not sure if SvelteKit offers something similar to Next.js' middleware, but one pattern you could try replicate would be to re-route users to a mobile view/route based on the incoming user-agent headers (recommended to use a package like https://www.npmjs.com/package/ua-parser-js with this approach)
